test/e2e/common_test.go

```diff
@@ -138,16 +138,7 @@ const (
 )
 
 var _ = SynchronizedBeforeSuite(func() []byte {
-	// One global scratch directory under which all test files will live.
-	// The usual case is that these tests are running in CI, on VMs
-	// with limited RAM, so we use /var/tmp.
-	baseTmpDir := "/var/tmp"
-	if os.Getenv("CI") == "" {
-		// Almost certainly a manual run, e.g., a developer with
-		// a hotrod workstation. Assume they know what they're doing.
-		baseTmpDir = ""
-	}
-	globalTmpDir, err := os.MkdirTemp(baseTmpDir, "podman-e2e-")
+	globalTmpDir, err := os.MkdirTemp("", "podman-e2e-")
```

Comment:
No, that wasn't my intention. I just meant to say, this is kind of tricky.

For WIP purposes, I think it would be fine to hardcode `/tmp` and just get a sense for how things go.

For longer-term purposes, if you determine that tmpfs works well, then (I dunno. Problem deferred.)

Reply:
If you like to have tests no hard code path assumptions (which I agree with) we can mount a tmpfs at a random location in CI setup and point TMPDIR var there. This should ensure no assumptions can be made.